### PR TITLE
fix(actions): print repositories in alphabetical order on git tag

### LINKED ISSUE #38: git tag should sort repo list
makes it hard to know if you've left a repo out or not

### PR DESCRIPTION
**Pull request by arschles**
# Associated [Documentation](https://github.com/deis/workflow) PR(s)

This patch changes an implementation detail but no external interface details, so no documentation changes are needed.

Fixes #38

### CODE REVIEW

File: actions/repo_and_sha_list_test.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+func TestRepoAndShaListSort(t *testing.T) {
+	rasl := newEmptyRepoAndShaList()
+	rasl.Add(ras1)
+	rasl.Add(ras2)
+	rasl.Sort()
+	assert.Equal(t, len(rasl.repoNames), 2, "length of repo names list")
+	assert.Equal(t, rasl.repoNames[0], ras1.repoName, "name of first repo")
+	assert.Equal(t, rasl.repoNames[1], ras2.repoName, "name of second repo")
+}
```
Comment (vdice) [inline actions/repo_and_sha_list_test.go]: For über thoroughness, we could write a `TestRepoAndShaListString` test verifying the formatting of that method (even though base `repoAndSha.String()` nicely tested elsewhere...)

**Comment by vdice** [comment]
Other than a test suggestion, LGTM!

File: actions/repo_and_sha_list_test.go (hunk 1 above)
Comment (arschles) [inline actions/repo_and_sha_list_test.go]: @vdice good call. added the new test in https://github.com/deis/deisrel/pull/80/commits/1a59c183d0b57c06714d8877678fc2fd1f29c1a9

Comment (vdice) [inline actions/repo_and_sha_list_test.go]: looks great @arschles -- still LGTM of course... but, like.. even more LGTM-ier

**Pull request merged**